app/controllers/actions/JourneyAction.scala

```diff
@@ -41,8 +40,8 @@ case class JourneyAction @Inject()(cacheService: ExportsCacheService, mcc: Messa
     implicit val hc: HeaderCarrier =
       HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))
 
-    cacheService.get(request.session.data("sessionId")).map(_.map(_.choice)).map {
-      case Some(journeyType) => Right(JourneyRequest(request, Choice(journeyType)))
+    cacheService.get(request.sessionId).map {
+      case Some(cacheModel) => Right(JourneyRequest(request, cacheModel))
       case _ =>
         logger.error(s"Could not obtain journey type for ${eoriCacheId()(request)}")
         Left(Conflict("Could not obtain information about journey type"))
```

Comment:
I know you didnt change this, but why on earth do we return a Conflict here....

It should be a redirect to the start page or something, but not a Conflict!

Reply:
Magic 409 😂